mlflow/recipes/utils/wrapped_recipe_model.py

```diff
@@ -0,0 +1,35 @@
+import mlflow
+from mlflow.pyfunc import PythonModel
+import pandas as pd
+import numpy as np
+
+
+class WrappedRecipeModel(PythonModel):
+    def __init__(self, predict_scores_for_all_classes=True, prefix="predicted_"):
+        super().__init__()
+        self.predict_scores_for_all_classes = predict_scores_for_all_classes
+        self.prefix = prefix
```

Comment:
Is this `prefix` the same of `predict_prefix` in the step config? Prefer to have the same name to avoid confusion.